ecs-init/docker/docker.go

```diff
@@ -108,15 +108,17 @@ const (
 	iptablesLib64Dir    = "/lib64"
 	iptablesUsrLib64Dir = "/usr/lib64"
 
-	hostCapabilitiesResourcesRootDir      = "/var/lib/ecs"
+	hostCapabilitiesResourcesRootDir      = "/var/lib/ecs/deps"
 	containerCapabilitiesResourcesRootDir = "/capabilities"
 
-	capabilityExecName                       = "exec"
-	capabilityExecHostBinRelativePath        = "ssm-agent/linux_amd64"
-	capabilityExecContainerBinRelativePath   = "bin"
-	capabilityExecHostCertsDir               = "/etc/pki/ca-trust/extracted/pem"
-	capabilityExecContainerCertsRelativePath = "certs"
-	capabilityExecRequiredCert               = "tls-ca-bundle.pem"
+	capabilityExecName                        = "execute-command"
+	capabilityExecHostBinRelativePath         = "bin"
+	capabilityExecContainerBinRelativePath    = "bin"
+	capabilityExecHostConfigRelativePath      = "config"
+	capabilityExecContainerConfigRelativePath = "config"
+	capabilityExecHostCertsDir                = "/etc/pki/ca-trust/extracted/pem"
```

Comment:
this will change too, if we are copying instance certs to exec deps folder.

Reply:
thanks for pointing it out! discussed offline, will keep this instance cert path for now; we'll change later once we change to the new model (copying instance certs to the exec deps folder)